src/core/lib/security/authorization/cel_evaluation_engine.h

```diff
@@ -0,0 +1,66 @@
+
+// Copyright 2020 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef GRPC_CORE_LIB_SECURITY_AUTHORIZATION_CEL_EVALUATION_ENGINE_H
+#define GRPC_CORE_LIB_SECURITY_AUTHORIZATION_CEL_EVALUATION_ENGINE_H
+
+#include <grpc/support/port_platform.h>
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "src/core/ext/upb-generated/envoy/config/rbac/v2/rbac.upb.h"
+#include "src/core/ext/upb-generated/google/api/expr/v1alpha1/syntax.upb.h"
+#include "upb/upb.hpp"
+
+// CelEvaluationEngine makes an AuthorizationDecision to ALLOW or DENY the
+// current action based on the condition fields in two provided RBAC policies.
```

Comment:
The comment seems confusing. 
.. based on condition fields in provided RBAC policies
Also update the example below.